### PR TITLE
Use software.repos.intel.com Conda Channel

### PR DESCRIPTION
**Pull request by tylertitsworth**
## Description
anaconda.com/intel no longer exists, use the new channel url.

## Related Issue
n/a

## Changes Made

- [x] The code follows the project's [coding standards](../CONTRIBUTING.md#code-style).
- [x] No Intel Internal IP is present within the changes.
- [x] The documentation has been updated to reflect any changes in functionality.

## Validation

- [x] I have tested any changes in container groups locally with [`test_runner.py`](../test-runner/README.md) with all existing tests passing, and I have added new tests where applicable.

### CODE REVIEW

**Review by sramakintel — DISMISSED** [review]
looks good.

File: preset/deep-learning/Dockerfile [hunk 1]
```diff
@@ -172,7 +172,7 @@ ARG TORCHAUDIO_CPU_VERSION
 ARG DEEPSPEED_VERSION
 
 # PyTorch CPU Env - conda packages
-RUN conda create -yn pytorch-cpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda create -yn pytorch-cpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```
Comment (jafraustro) [inline preset/deep-learning/Dockerfile:175]: `intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-cpu -c "${INTEL_CHANNEL}" -c conda-forge \`

File: preset/deep-learning/Dockerfile [hunk 2]
```diff
@@ -227,7 +227,7 @@ ARG IDEX_VERSION
 ARG DEEPSPEED_VERSION
 
 # PyTorch GPU Env - conda packages
-RUN conda create -yn pytorch-gpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda create -yn pytorch-gpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```
Comment (jafraustro) [inline preset/deep-learning/Dockerfile:230]: `intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-gpu -c "${INTEL_CHANNEL}" -c conda-forge \`

File: preset/deep-learning/Dockerfile [hunk 3]
```diff
@@ -429,11 +429,11 @@ ENV UCX_TLS=ud,sm,self
 
 USER dev
 
-RUN conda install -n pytorch-cpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda install -n pytorch-cpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```
Comment (jafraustro) [inline preset/deep-learning/Dockerfile:432]: `intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-cpu -c "${INTEL_CHANNEL}" -c conda-forge \`

File: preset/deep-learning/Dockerfile [hunk 4]
```diff
@@ -429,11 +429,11 @@ ENV UCX_TLS=ud,sm,self
...
         deepspeed="${DEEPSPEED_VERSION}" \
         'tensorboardx>=2.6.2.2'
 
-RUN conda install -n pytorch-gpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda install -n pytorch-gpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```
Comment (jafraustro) [inline preset/deep-learning/Dockerfile:436]: `intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-cpu -c "${INTEL_CHANNEL}" -c conda-forge \`

**Review by jafraustro — CHANGES_REQUESTED** [review]

**Review by jafraustro — DISMISSED** [review]
LGTM

**Pull request merged**